app/models/dimensions/edition.rb

```diff
@@ -25,11 +25,11 @@ class Dimensions::Edition < ApplicationRecord
 
   def self.search(query)
     sql = <<~SQL
-      to_tsvector('english',title) @@ plainto_tsquery('english', :q) or
-      to_tsvector('english'::regconfig, replace((base_path)::text, '/'::text, ' '::text)) @@ plainto_tsquery('english', :q2)
+      to_tsvector('english',title) @@ plainto_tsquery('english', :search_term) or
+      to_tsvector('english'::regconfig, replace((base_path)::text, '/'::text, ' '::text)) @@ plainto_tsquery('english', :title_url_2)
     SQL
 
-    where(sql, q: query, q2: query.tr('/', ' '))
+    where(sql, search_term: query, title_url_2: query.tr('/', ' '))
```

Comment:
Maybe use `search_term_without_slash` instead of `title_url_2` here, as I think that would be clearer.

Reply:
Thank you. I have addressed your comment 👍